### PR TITLE
atuin 0.7.1 (new formula)

### PR DESCRIPTION
**Pull request by iloveitaly**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Review by nandahkrishna — DISMISSED** [review]
LGTM. Thanks, @iloveitaly!

I'll merge this in a while if nobody else has any comments.

File: Formula/atuin.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "autoload -U add-zsh-hook", shell_output("atuin init zsh")
+    assert_match "", shell_output("atuin history list")
```
Comment (SMillerDev) [inline Formula/atuin.rb:16]: This doesn't actually check anything

Comment (nandahkrishna) [reply Formula/atuin.rb:16]: Oops, missed that. This might work:

```suggestion
    assert shell_output("atuin history list").blank?
```

Comment (iloveitaly) [reply Formula/atuin.rb]: Yup, this looks like what I was missing!

**Review by SMillerDev — APPROVED** [review]

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**